pyartcd/pyartcd/pipelines/build_sync.py

```diff
@@ -255,7 +259,15 @@ async def _update_nightly_imagestreams(self):
             cmd.extend([f'--exclude-arch {arch}' for arch in self.exclude_arches])
         if self.runtime.dry_run:
             cmd.extend(['--skip-gc-tagging', '--moist-run'])
-        await exectools.cmd_assert_async(cmd)
+
+        rc, stdout, stderr = await exectools.cmd_gather_async(cmd)
```

Comment:
Running `cmd_gather_async` like this will capture stdout and stderr, and we won't be able to see them in the logs.
We currently need to capture stderr to check for viability, so we could change this into something like:
```
rc, _, stderr = await exectools.cmd_gather_async(cmd, stderr=None)
self.runtime.logger.info('doozer release:gen-payload output:\n%s, stderr)
```

However, we won't see any log until the process is completed, then get a full dump which doesn't quite help to understand what's going on. What we can do instead, is change [this line](https://github.com/openshift-eng/art-tools/blob/main/doozer/doozerlib/cli/release_gen_payload.py#L300) in doozer and have that message sent to stdout. Then, pyartcd would do something like:
```
rc, stdout, _ = await exectools.cmd_gather_async(cmd, stdout=None)
self.runtime.logger.info('doozer release:gen-payload output:\n%s, stdout)